tests/wordcount.js

```diff
@@ -41,13 +44,13 @@ describe( 'WordCount', () => {
 			expect( wordCountPlugin.characters ).to.equal( 0 );
 		} );
 
-		it( 'has defined "_outputView" property', () => {
-			expect( wordCountPlugin._outputView ).to.be.undefined;
-		} );
-
 		it( 'has "WordCount" plugin name', () => {
 			expect( WordCount.pluginName ).to.equal( 'WordCount' );
 		} );
+
+		it( 'has define "_config" object', () => {
```

Comment:
I don't think we need to check for private properties.

Reply:
I removed this test